### PR TITLE
Pretty-printing DNSSEC RR

### PR DESCRIPTION
**Pull request by kazu-yamamoto**
The subject says all.

### CODE REVIEW

**Comment by kazu-yamamoto** [comment]
- `dug jprs.jp dnskey`
- `dug www.cloudflare.com dnskey`

File: dnsext-dnssec/DNS/SEC/PubKey.hs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+import DNS.Types
+import DNS.Types.Internal
+import qualified DNS.Types.Opaque as Opaque
+
+import DNS.SEC.Imports
+import DNS.SEC.PubAlg
+
+data PubKey = PubKey_RSA { rsa_size :: Int, rsa_e :: Opaque, rsa_n :: Opaque }
+            | PubKey_ECDSA { dsa_x :: Opaque, dsa_y :: Opaque }
```
Comment (khibino) [inline dnsext-dnssec/DNS/SEC/PubKey.hs:11]: I think `ec_x` and `ec_y` are clearer names. what do you think?

**Comment by khibino** [comment]
* `dug ed25519.no. dnskey`
* `dug ed448.no. dnskey`

File: dnsext-dnssec/DNS/SEC/PubKey.hs (hunk 1 above)
Comment (kazu-yamamoto) [reply dnsext-dnssec/DNS/SEC/PubKey.hs:11]: How about `ecdsa_x` and `ecdsa_y`?

Comment (khibino) [reply dnsext-dnssec/DNS/SEC/PubKey.hs:11]: I think that's good too

**Review by khibino — APPROVED** [review]

**Comment by kazu-yamamoto** [comment]
@khibino For X25519 and X448, `PubKey_Opaque` is good enough, right?

**Comment by khibino** [comment]
Yes, I think that's good.

**Comment by kazu-yamamoto** [comment]
Rebased and merged.Thanks!

**Pull request merged**